pkgs/build-support/build-dotnet-module/default.nix

```diff
@@ -21,11 +21,22 @@
 # Unfortunately, dotnet has no method for doing this automatically.
 # If unset, all executables in the projects root will get installed. This may cause bloat!
 , executables ? null
+# Pack module as nupkg and push it to `$out/share`. Disabled by default
```

Comment:
Please update the nixpkgs manual on buildDotnetModule to document this new option.

Reply:
Added in the updated commit